### DISCUSSION TITLE
Navbar overlaps on scroll with other elements.

### DISCUSSION THREAD

**Discussion by gajrajgchouhan** [fastapi/fastapi#6042] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
--
```

### Description

Currently, the label of the navbar on right doesn't have background so it overlaps with other element when scrolling. This can be fixed by adding background-color and z-index.

```css
    background-color: white;
    z-index: 5;
```

```html
<label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
```

Before :
![Screenshot 2022-04-28 113012](https://user-images.githubusercontent.com/54475046/165687055-37600901-4e11-4eb2-966e-42d2d0232c72.png)
After :
![Screenshot 2022-04-28 113032](https://user-images.githubusercontent.com/54475046/165687060-45196dab-bf1c-4c5d-9e41-26cbb4265b19.png)

### Operating System

Windows

### FastAPI Version

-

### Python Version

-

**Comment by JarroVGIT** [comment]
I cannot reproduce, this might have been fixed already.
